### DISCUSSION TITLE
Existing assets in S3 Bucket?

### DISCUSSION THREAD

**Discussion by edsel45** [pimcore/pimcore#16473] [Q&A / Support]
Is there a way to get Pimcore to recognize assets in an existing S3 bucket when connecting with Flysystem? Am I missing something here?

**Comment by kingjia90** [comment]
Not sure, but i guess there are no out-of-the-box commands for that.

What about a script that read the folder content (in S3 bucket) and triggers a creation/save via [PHP API](https://pimcore.com/docs/platform/Pimcore/Assets/Working_with_PHP_API/) for each file after parsing the folder structure? 
May not have the creation dates though.
